CONTRIBUTING.md

```diff
@@ -85,6 +85,36 @@ This will setup your project along with firebase emulator in a docker environmen
 
 You should fill in these values in their relevant fields in the `.env` file.
 
+
+### Firebase Emulator Setup
+
+From your parent directly run cd/functions
+1. run 
+```
+shell npm init
+ ```
+2. refer this site [https://firebase.google.com/docs/emulator-suite/install_and_configure]
+3. make sure you have the correct jdk version installed
+4. then go to paren directory bu running command (```cd ..```)
+5. now lets connect your local firebase to cloub by running command (```firebase login```)
+6. then authenticate your firebase using browser and set the selected web app for codelabz
+7. then run the command (```firebase init```)
+8. select all the emulator necessitites by pressing a or selecting them amnually and pressing space
+9. answer the commands
+10. lets set up your credentials of test data
+11. then run command
+```
+firebase emulators:start --import=testdata
+```
+12. if data is already present then run run
```

Comment:
Can you change this step to "If you want to start without any testdata, use the following command"